cmd/helm/install.go

```diff
@@ -388,3 +397,19 @@ func defaultNamespace() string {
 	}
 	return "default"
 }
+
+func (i *installCmd) checkDependencies(ch *chart.Chart, reqs *chartutil.Requirements) {
+	deps := ch.GetDependencies()
```

Comment:
If you created a function `func checkDependencies(ch *chart.Chart, reqs *chartutil.Requirements, out io.Writer)`, could you combine the three different versions of this function?